config/src/test/java/org/axonframework/config/utils/AssertUtils.java

```diff
@@ -30,20 +32,19 @@ private AssertUtils() {
     /**
      * Assert that the given {@code assertion} succeeds with the given {@code time} and {@code unit}.
      *
-     * @param time      an {@code int} which paired with the {@code unit} specifies the time in which the assertion must
+     * @param duration  specifies the time in which the assertion must
      *                  pass
-     * @param unit      a {@link TimeUnit} in which {@code time} is expressed
      * @param assertion a {@link Runnable} containing the assertion to succeed within the deadline
      */
     @SuppressWarnings("Duplicates")
-    public static void assertWithin(int time, TimeUnit unit, Runnable assertion) {
+    public static void assertRetryingWithin(Duration duration, Runnable assertion) {
         long now = System.currentTimeMillis();
-        long deadline = now + unit.toMillis(time);
+        long deadline = now + duration.toMillis();
         do {
             try {
                 assertion.run();
                 break;
-            } catch (AssertionError e) {
+            } catch (AssertionFailedError e) {
```

Comment:
Yes, for JUnit5.
I think there's no necessity to keep `AssertionError`because this `AssertUtils` class is replaced in the config module and used only in it (`DefaultConfigurerTest`). And tests in this module is migrated to JUnit5 in this PR.